Achievements/crypto-guru.md

```diff
@@ -0,0 +1,27 @@
+# Crypto Guru Achievement
+
+# Goal: 
+
+Demonstrate your mastery of crypto protocols by notarizing your player-space's README.md file on a blockchain.
```

Comment:
OOH! OOH! We can do this with CircuitBuilder! In fact, it is one of our built-in templates!